products/transcription/accuracy-benchmarks.mdx

```diff
@@ -0,0 +1,152 @@
+
+---
+
+## German
+
+| Dataset     | Sub-Dataset                     | Accuracy | WER  | Source                                           |
+| ----------- | ------------------------------- | -------- | ---- | ------------------------------------------------ |
+| CommonVoice | cv-corpus-13.0-delta-2023-03-09 | 96.3%    | 3.7% | [Common Voice](https://commonvoice.mozilla.org/) |
+
+---
+
+## Hindi
+
+| Dataset     | Sub-Dataset               | Accuracy | WER   | Source                                           |
+| ----------- | ------------------------- | -------- | ----- | ------------------------------------------------ |
+| CommonVoice | cv-corpus-20.0-2024-12-06 | 84.0%    | 16.0% | [Common Voice](https://commonvoice.mozilla.org/) |
+
+---
+
+## Italian
+
+| Dataset     | Sub-Dataset | Accuracy | WER  | Source                                           |
+| ----------- | ----------- | -------- | ---- | ------------------------------------------------ |
+| CommonVoice | —           | 93.3%    | 6.7% | [Common Voice](https://commonvoice.mozilla.org/) |
+
+---
+
+## Russian
+
+| Dataset     | Sub-Dataset | Accuracy | WER  | Source                                           |
+| ----------- | ----------- | -------- | ---- | ------------------------------------------------ |
+| CommonVoice | —           | 96.4%    | 3.6% | [Common Voice](https://commonvoice.mozilla.org/) |
+
+---
+
+## Hebrew
+
+| Dataset     | Sub-Dataset               | Accuracy | WER   | Source                                           |
+| ----------- | ------------------------- | -------- | ----- | ------------------------------------------------ |
+| CommonVoice | cv-corpus-17.0-2024-03-15 | 84.2%    | 15.8% | [Common Voice](https://commonvoice.mozilla.org/) |
+
+---
+
+## Kazakh
+
+| Dataset     | Sub-Dataset               | Accuracy | WER   | Source                                           |
+| ----------- | ------------------------- | -------- | ----- | ------------------------------------------------ |
+| CommonVoice | cv-corpus-19.0-2024-09-13 | 51.0%    | 49.0% | [Common Voice](https://commonvoice.mozilla.org/) |
+
+---
+
+## Urdu
+
+| Dataset     | Sub-Dataset              | Accuracy | WER   | Source                                           |
+| ----------- | ------------------------ | -------- | ----- | ------------------------------------------------ |
+| CommonVoice | cv-corpus-9.0-2022-04-27 | 78.8%    | 21.2% | [Common Voice](https://commonvoice.mozilla.org/) |
+
+---
+
+## Thai _(in progress)_
+
+| Dataset     | Sub-Dataset                     | Accuracy | WER     | Source                                           |
+| ----------- | ------------------------------- | -------- | ------- | ------------------------------------------------ |
+| CommonVoice | cv-corpus-10.0-delta-2022-07-04 | 33.0%    | 67.0%\* | [Common Voice](https://commonvoice.mozilla.org/) |
+
+## _Note: Thai WER may need a recalculation due to formatting issues._
```

Comment:
Should this be more indented, or maybe even just a [callout box](https://mintlify.com/docs/content/components/callouts) of some kind? It looks kind of odd in the Table of Contents